### ISSUE TITLE
Interfaz muy pequeña en mi opnion

### ISSUE THREAD

**Issue by Mrgaton** [JSPaste/JSP-Frontend#55]
como en fnlb y en muchas otras en mi opinion creo que esta todo muy pequeño

si le hago zoom a 125% opino que queraria perfecto si fuera asi por default

**Comment by inetol** [comment]
Como uso un tamaño de interfaz idéntico para el IDE no siento que haya problema alguno, pero si es verdad que en pantallas ultra-wide, 4K o superior todo se hace demasiado pequeño para ser legible sin tener que usar el zoom del navegador.

Se puede hacer la interfaz de un tamaño constante para cualquier pantalla, pero no sé si se joderá en teléfonos

**Comment by tnfAngel** [comment]
En el caso de telefonos se puede simpemente desactivar y dejarlo como está ahora, pero como se haría de tamaño constante (sin joder lo demás)?

**Comment by tnfAngel** [comment]
supongo que jugando dinámicamente con el initial-scale https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag

**Comment by tnfAngel** [comment]
igualmente creo que surgiría un problema luego de la hidratación (ya que en el SSR no se tiene en cuenta el tamaño de la pantalla), en pantallas de alta resolución, causará un zoom inesperado luego de la hidratación, aunque creo que es asumible

**Comment by inetol** [comment]
> igualmente creo que surgiría un problema luego de la hidratación (ya que en el SSR no se tiene en cuenta el tamaño de la pantalla), en pantallas de alta resolución, causará un zoom inesperado luego de la hidratación, aunque creo que es asumible

El SSR no estaba deshabilitado hasta que cargabas el componente con dynamic o lazy?

**Comment by tnfAngel** [comment]
No, se usa SSR incluso teniendo eso desactivado, si te fijas hay incluso un layout shift en el navbar/information (links de la derecha o el texto del lenguaje) por el mobile-first cuando carga la página (luego de la hidratación obtene el tamaño de la pantalla y se ajusta)

**Comment by inetol** [comment]
@Mrgaton Estoy pensando y preparando las bases para el rework de la interfaz donde solucionaré esto. Para complementar este issue podría añadir la captura del zoom al propio editor, cuando se hiciese zoom solo cambie el tamaño del editor y no de la interfaz, ya que esta última ya sería reactiva con el tamaño del viewport.

A eso añadir también los controles dedicados en la configuración, algo de este palo:
![Image](https://github.com/user-attachments/assets/183c5554-9aae-455a-8f70-ac20e881dca3)
